package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "test": "mocha -r ts-node/register test/libraryUtilitiesTest.ts",
```

Comment:
@monikaprabhu do you mean we now have `tests` and `mocks` folders instead of `__tests__` and `__mocks__`?